# On-boarding

Hello and welcome!  
This document is a collection of the tools and tasks that the lab is interested in. 

# Databases and repositories  
## Gene expression data 
Databases for gene expression data in raw and parsed form. 

[GEO](https://www.ncbi.nlm.nih.gov/geo/) 

SRA: https://www.ncbi.nlm.nih.gov/sra

ArrayExpress: https://www.ebi.ac.uk/arrayexpress/ 

ENA: https://www.ebi.ac.uk/ena

Single Cell Expression Atlas: https://www.ebi.ac.uk/gxa/sc/home

Expression Atlas: https://www.ebi.ac.uk/gxa/home

Allen Brain Atlas: https://portal.brain-map.org/

GEMMA: https://gemma.msl.ubc.ca/

## Annotations
GENCODE: https://www.gencodegenes.org/

Human: https://www.gencodegenes.org/human/

Mouse: https://www.gencodegenes.org/mouse/

modENCODE (for fly and worm): http://www.modencode.org/ 

ENSEMBL: http://useast.ensembl.org/index.html  

Many species: http://useast.ensembl.org/info/about/species.html

â€ƒ
## Core datasets 
GTEx:  https://gtexportal.org/home/

GEUVADIS:  https://www.ebi.ac.uk/Tools/geuvadis-das/

ENCODE: https://www.encodeproject.org/ 

?? BrainSpan: https://www.brainspan.org/ 

## Processed expression data 
Recount2: https://jhubiostatistics.shinyapps.io/recount/ 

GEMMA: https://gemma.msl.ubc.ca/

ARCHS4: https://amp.pharm.mssm.edu/archs4/

## Co-expression databases 
COEXPRESdb:  https://coxpresdb.jp/

HumanBase: https://hb.flatironinstitute.org/ 


## Ontologies 
Gene Ontology
http://geneontology.org/ 
 
Related work: 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389513/

Human phenotype ontology (HPO): https://hpo.jax.org/app/

Cell http://www.obofoundry.org/ontology/cl.html

Experimental factor (EFO) https://www.ebi.ac.uk/efo/

Other 
https://amp.pharm.mssm.edu/Harmonizome/

https://github.com/OBOFoundry/purl.obolibrary.org/

## Pathways 
KEGG: https://www.genome.jp/kegg/

Reactome:  https://reactome.org/

Biocarta: (original site dead, but can be found in other places)  https://amp.pharm.mssm.edu/Harmonizome/dataset/Biocarta+Pathways


## Protein data 
biogrid: https://thebiogrid.org/ 

STRING: https://string-db.org/

I2D: http://ophid.utoronto.ca/ophidv2.204/ 

HIPPIE: http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/ 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210659/ 

https://www.ebi.ac.uk/interpro/

HuRI (human reference or CCSB) http://interactome.baderlab.org/download 

Human Protein Atlas: https://www.proteinatlas.org/

## Genomic data
1000 genomes: https://www.internationalgenome.org/

ExAC: http://exac.broadinstitute.org/

gnoMAD: https://gnomad.broadinstitute.org/

TOPmed: https://www.nhlbiwgs.org/
(can be accessed here:) https://bravo.sph.umich.edu/freeze5/hg38/

dbGAP: https://www.ncbi.nlm.nih.gov/gap/


### Key tools
GATK: https://software.broadinstitute.org/gatk/

Samtools:  http://www.htslib.org/ 

Tabix: http://www.htslib.org/doc/tabix.html

IGVtools: https://software.broadinstitute.org/software/igv/igvtools 


# Gene expression analysis
## Microarray
https://wiki.bits.vib.be/index.php/Analyze_your_own_microarray_data_in_R/Bioconductor 

## RNA-sequencing 
https://bioinformatics-core-shared-training.github.io/RNAseq-R/

https://www.bioconductor.org/packages/devel/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html

### Bulk 
https://en.wikipedia.org/wiki/List_of_RNA-Seq_bioinformatics_tools 
fastX: http://hannonlab.cshl.edu/fastx_toolkit/ 
fastQC: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ 

### Single-cell 
https://www.scrna-tools.org 

### Other 
Biojupies https://amp.pharm.mssm.edu/biojupies/ 

# Gene set enrichment 
## Key tools 
ermineJ https://erminej.msl.ubc.ca/ 

GSEA http://software.broadinstitute.org/gsea/index.jsp 

DAVID https://david.ncifcrf.gov/ 

GEO2Enrichr https://amp.pharm.mssm.edu/g2e/

# Alignment tools 
## STAR
https://github.com/alexdobin/STAR

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631051/

https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf

## Kallisto
https://pachterlab.github.io/kallisto/

https://www.nature.com/articles/nbt.3519

https://pachterlab.github.io/kallisto/starting

## Salmon
https://combine-lab.github.io/salmon/

https://www.nature.com/articles/nmeth.4197

https://combine-lab.github.io/salmon/getting_started/

## Alevin
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1670-y

https://salmon.readthedocs.io/en/latest/alevin.html

## Bowtie2
http://bowtie-bio.sourceforge.net/bowtie2/index.shtml

https://academic.oup.com/bioinformatics/article/35/3/421/5055585

https://www.nature.com/articles/nmeth.1923

https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-3-r25

http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml


# Model organisms 
## Orthology 
Homologene: https://www.ncbi.nlm.nih.gov/homologene 

OrthoDB: https://www.orthodb.org/

BUSCO genes: https://busco.ezlab.org/  

Papers
https://www.annualreviews.org/doi/full/10.1146/annurev.genet.39.073003.114725?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub%3Dpubmed

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC138949/



# Lab bits and pieces  
https://gillislab.github.io/publications/

https://gillislab.github.io/background/ 



